### DISCUSSION TITLE
404 error Help please!

### DISCUSSION THREAD

**Discussion by jinlee487** [community/community#101157] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, my gitpages https://jleecode.dev/ is not working. I have an active gitpages for a year now until today I discovered that it stopped working. I deployed it last 2 weeks ago and was working fine. Today I tried redeploying my app by pushing to main but the none of the auto deploy git actions features pages-build-deployment did not start. I then tried manual rerunning the last deployment from 2 weeks ago but the deployment failed with the following results. 
<img width="683" alt="image" src="https://github.com/community/community/assets/46912607/79e97100-b4e0-4978-9dd8-0543f59e19bf">

report response
Run gh api -X POST "repos/$GITHUB_REPOSITORY/pages/telemetry" \
gh: page is disabled (HTTP 404)
{"message":"page is disabled","documentation_url":"https://docs.github.com/rest/pages/pages#report-telemetry-back-to-datadog"}
Error: Process completed with exit code 1.

deploy response
Error: Creating Pages deployment failed
Error: HttpError: Validation Failed

The build is working fine so I dont think anything is wrong with the code. As two weeks ago, I verified that the site was working fine after deployment. Could this be a server issue? the gitstatus is showing https://www.githubstatus.com/ green for everything as of now. And other users gitpages are all working fine. What could be the issue? help please.

**Comment by fury-05** [comment, marked as answer]
Check once the page setting by going to repository and then setting then page, My guess is page setting have be affected some how.

If you see all ok their then might be some different problem, as you said build and code are not affected checking page setting you give you idea what is causing error.

**Reply by jinlee487** [reply]
my accounts student pro has ended so I had to revert my gitpages repo from prviate to public.
